Userland/Libraries/LibFileSystemAccessClient/Client.cpp

```diff
@@ -48,6 +48,31 @@ DeprecatedResult Client::try_request_file_read_only_approved_deprecated(GUI::Win
     return handle_promise<DeprecatedResult>(id);
 }
 
+Result Client::request_file_read_only_approved(GUI::Window* parent_window, DeprecatedString const& path)
```

Comment:
The commit title doesn't fit the standards for it being on one line. It's annoying with LibFileSystemAccessClient being so long, but maybe something like `LibFileSystemAccessClient: Add functions returning FSAC::File`?